solarSystem.scss

```diff
@@ -0,0 +1,142 @@
+@media screen and (min-width: 600px) {
+  body {
+    background-image: url("cosmos-1853491_1920.jpg");
+  }
+
+  .planet-card-container {
+    display: none;
+    border: white solid 1px;
+  }
+
+  .title-image-container {
```

Comment:
there is slight overlap of the title image out of the title container just before the breakpoint - update so title image shrinks accordingly